clients/ui/frontend/src/shared/api/apiUtils.ts

```diff
@@ -76,7 +76,10 @@ const callRestJSON = <T>(
 
   return fetch(`${host}${path}${searchParams ? `?${searchParams}` : ''}`, {
```

Comment:
I think it's generally better to use the URL api to construct this rather than a string template. You get URL encoding / escaping for free that way.

That said this is a nit so I won't hold the PR up for it :)